REVERSE_LC.php

```diff
@@ -457,29 +450,42 @@ public function getRecords($project_id = false) {
 		}
 		return $this->records;
 	}
-	public function getMySiteData() {
+
+	public function getAccessLevelByRole($userRole)
+	{
+		$project_id  = $this->getProjectId();
+		$roles       = $this->getProjectSetting("access_level_field_value",$project_id)[0];
+		$level       = $this->getProjectSetting("access_level",$project_id)[0];
+		$accessLevel = [];
+		foreach($roles as $seq => $role) {
```

Comment:
I see that you're pulling the first array element for $roles and $level. I don't think that's going to be an array even if access_level_field_value is a repeating setting, so I don't think this foreach is going to work.

Reply:
Sorry, following up, I missed how the settings had been changed. Let me review this again before you do anything.

Reply:
I moved the settings at the root level